img/logo/logo.svg

```diff
@@ -0,0 +1,6 @@
 70.5968 24.2772 69.4546 23.2339C68.5041 22.3757 67.3284 21.9456 65.9349 21.9456C63.6517 21.9456 62.0638 22.9624 61.1787 24.9926C60.8588 25.7259 60.6997 26.6108 60.6997 27.6427C60.6997 28.6816 60.8588 29.569 61.1787 30.2992C61.4932 31.0361 61.901 31.6233 62.3985 32.0775V32.0773ZM83.4205 19.3384C85.8725 19.3384 87.8381 20.1034 89.3179 21.6307C90.7996 23.18 91.5379 25.1913 91.5379 27.6635C91.5379 30.1314 90.7996 32.1452 89.3179 33.6935C87.8381 35.2228 85.8725 35.9846 83.4205 35.9846C80.946 35.9846 78.981 35.2228 77.522 33.6935C76.0414 32.1669 75.302 30.1524 75.302 27.6635C75.302 25.1703 76.0414 23.1595 77.522 21.6307C78.981 20.1034 80.946 19.3384 83.4205 19.3384ZM81.5654 33.0661C80.9416 32.8574 80.3795 32.5276 79.8846 32.0773V32.0775C79.3853 31.6233 78.9808 31.0361 78.6629 30.2992C78.3435 29.569 78.1866 28.6816 78.1866 27.6427C78.1866 26.6108 78.3435 25.7259 78.6629 24.9926C79.5513 22.9624 81.1374 21.9456 83.4203 21.9456C84.8131 21.9456 85.9885 22.3757 86.9383 23.2339C88.0802 24.2772 88.6507 25.746 88.6507 27.6278C88.6507 29.5747 88.0802 31.0678 86.9383 32.0936C86.4318 32.5291 85.8642 32.8574 85.2428 33.0661C84.6199 33.2765 84.0074 33.3809 83.4037 33.3809C82.7996 33.3809 82.1871 33.2765 81.5654 33.0661ZM138.607 24.2939C138.319 24.8768 138.107 25.4109 138.007 26.1379H146.968C146.747 24.69 146.2 23.6426 145.371 22.8792C144.59 22.1622 143.609 21.9452 142.486 21.9452C140.699 21.9452 139.353 22.7894 138.607 24.2939ZM161.458 25.4666C161.33 23.5045 159.84 22.5936 159.84 22.5936C159.84 22.5936 158.827 21.9155 157.354 21.9155C156.166 21.9155 155.035 22.1868 154.403 22.8644C153.797 23.5149 153.38 23.5149 152.14 23.5147L152.079 23.5147C151.292 23.5147 151.341 22.7324 151.479 22.4697C153.173 19.24 157.865 19.3462 157.865 19.3462V19.3441C160.185 19.3441 162.021 20.4648 162.761 21.2971C163.793 22.4338 164.309 23.7892 164.309 25.4909V33.3129V34.4127V35.1525C164.309 35.6122 163.946 35.9846 163.494 35.9846H162.301C161.855 35.9846 161.489 35.6122 161.489 35.1525V34.4127V33.8587L161.472 33.8732C160.809 34.4425 159.014 35.9846 155.665 35.9846C154.242 35.9846 153.148 35.5333 152.197 34.6952C151.205 33.8107 150.615 32.6868 150.615 31.2562C150.615 29.7031 151.193 28.486 152.351 27.5989C153.608 26.635 155.405 26.1537 157.74 26.1537H160.973C161.452 26.1537 161.471 25.5992 161.458 25.4666ZM161.478 30.2953L161.478 30.3055C161.475 30.4111 161.465 30.7207 161.283 30.9714C160.857 31.573 160.343 32.094 159.674 32.4812C158.907 32.9228 157.914 33.3806 157.026 33.3806C155.02 33.3806 153.756 32.5442 153.815 30.7751C153.845 29.8814 154.485 29.3609 155.239 29.0147C155.994 28.671 157.046 28.4949 158.389 28.4949H161.089C161.089 28.4949 161.478 28.4865 161.478 28.8619V30.2953ZM130.803 30.2101C130.029 32.3436 128.482 33.4103 126.16 33.4103C125.117 33.4103 121.418 32.9722 121.418 27.6902C121.418 23.3316 124.419 22.1245 126.16 22.1245C127.666 22.1245 128.87 22.5963 129.763 23.5367C130.701 24.5579 131.171 25.9694 131.171 27.7676C131.171 28.6861 131.048 29.5 130.803 30.2101Z" fill="#00343D"/>
```

Comment:
These SVGs are not optimized, you can use these app
For SVG: https://imageoptim.com/mac
For PNG, JPEG and among others: https://tinypng.com/
@Jefer-Max